Apps/WebClient/src/Server/Services/UserEmailService.cs

```diff
@@ -235,7 +235,14 @@ private void AddVerificationEmail(string hdid, string toEmail, Guid inviteKey)
 
             this.messageVerificationDelegate.Insert(messageVerification);
 
-            this.emailQueueService.QueueNewEmail(messageVerification.Email);
+            if (isVerified)
+            {
+                this.ValidateEmail(hdid, inviteKey);
```

Comment:
Maybe we should update the DB model of the verification that implies that it was verified through the jwt field?

Reply:
That sounds good but I don't think it solves the main problem around the email record being there when no email was sent